sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -327,6 +325,17 @@ private void cleanUp(Person person) {
 			person.setBurialDate(null);
 			person.setBurialPlaceDescription(null);
 			person.setBurialConductor(null);
+
+		}if(person.getPresentCondition()==PresentCondition.UNKNOWN){
```

Comment:
You can simply extend the first line in this method like this:

`if (person.getPresentCondition() == null || person.getPresentCondition() == PresentCondition.ALIVE || person.getPresentCondition() == PresentCondition.UNKNOWN) {`

instead of adding all these new code lines.